js/samples/04.ai.d.chainedActions.listBot/src/prompts/summarize/skprompt.txt

```diff
@@ -53,7 +56,12 @@ Here's the users query:
 ```
 
 Use the template to create an adaptive card.
+Create a Container for each list.
+Replace the first TextBlock text in the Container with the List name.
+Replace the second TextBlock text in the Container with the list items.
```

Comment:
This is great for summarizing all the lists. What if the user only wants to summarize just one specific list. Does it still work as expected?

Reply:
It works when we provide the list name for the summary